plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/font-picker.spec.js

```diff
@@ -262,7 +262,13 @@ test.describe(
 			const fontPickers = assembler.locator(
 				'.woocommerce-customize-store_global-styles-variations_item'
 			);
-			await expect( fontPickers ).toHaveCount( 10 );
+
+			await expect
+				.poll( async () => {
+					const count = await fontPickers.count();
+					return count === 10;
+				} )
+				.toBeTruthy();
```

Comment:
The assertion here is indirect, we're asserting the result of comparison between the count and `10`, not asserting the count to be `10`, which won't give us a good error message when the test fails. What about asserting the count directly to `10`:
```js
await expect
    .poll(async () => await fontPickers.count())
    .toEqual(10);
```

Reply:
It makes sense! I updated it with a538061d664e989e94a6c832121641dc7a763015